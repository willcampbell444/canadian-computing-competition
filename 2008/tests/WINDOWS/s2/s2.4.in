1000
7654
9999
0
