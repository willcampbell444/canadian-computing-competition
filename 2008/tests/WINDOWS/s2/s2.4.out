3141549
184045801
314096101
